atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/ColumnSelection.java

```diff
@@ -44,12 +43,12 @@ private ColumnSelection(SortedSet<byte[]> selectedColumns) {
 
     public static ColumnSelection valueOf(String serialized) {
         Set<byte[]> columns = Sets.newTreeSet(UnsignedBytes.lexicographicalComparator());
-        for (String strColumn : serialized.split("\\s*,\\s*")) {
-            strColumn = strColumn.trim();
-            if (strColumn.equals("")) {
+        for (String columnString : serialized.split("\\s*,\\s*")) {
+            String trimmedColumnString = columnString.trim();
```

Comment:
...although I see that this loop is like 5 lines long, so I'm not too bothered in this case.